labs-ios-starter/Surveys/Model Controllers/TopicController.swift

```diff
@@ -15,19 +15,22 @@ class TopicController {
     lazy var baseURL = profileController.baseURL
 
     /// public getter for CONTEXTS
-    var contexts: [ContextObject]  {
+    var contexts: [ContextObject] {
         CONTEXTS
     }
+
     /// private setter for contexts
     private var CONTEXTS: [ContextObject] = []
     /// public getter for QUESTIONS
     var questions: [Question] {
         QUESTIONS
     }
+
     /// private setter for questions
     private var QUESTIONS: [Question] = []
```

Comment:
Does it make more sense to you to group them by type?
```suggestion
           /// public getter for CONTEXTS
        var contexts: [ContextObject]  {
            CONTEXTS
        }
        /// private setter for contexts
        private var CONTEXTS: [ContextObject] = []

        /// public getter for QUESTIONS
        var questions: [Question] {
            QUESTIONS
        }
        /// private setter for questions
        private var QUESTIONS: [Question] = []
```

Reply:
Yeah, it does. I'll accept the change. I'm using a swift formatter that will see that closing bracket and will automatically put a line after it.